src/ScalarMessaging.js

```diff
@@ -564,12 +564,27 @@ const onMessage = function(event) {
     });
 };
 
+let listenerCount = 0;
```

Comment:
Yeah, that's one of the reasons why I don't like the split between react-sdk/riot-web. It's very subtle. I think we ended up fixing this on MatrixClientPeg by promoting the module-level variable to be a global?